scripts/process.py

```diff
@@ -1,678 +0,0 @@
-survey_questions = config['database']['questions']
-
-# https://docs.google.com/a/okfn.org/spreadsheet/ccc?key=0Aon3JiuouxLUdEVHQ0c4RGlRWm9Gak54NGV0UlpfOGc&usp=drive_web#gid=0
-survey_datasets = config['database']['datasets']
-
-# https://docs.google.com/a/okfn.org/spreadsheet/ccc?key=0AqR8dXc6Ji4JdE1QUS1qNjhvRDJaQy1TbTdJZDMtNFE&usp=drive_web#gid=1
-survey_places = config['database']['places']
-
-entries_dest = os.path.join(datastore_config['location'], 'entries.csv')
-submissions_dest = os.path.join(datastore_config['location'], 'submissions.csv')
-questions_dest = os.path.join(datastore_config['location'], 'questions.csv')
-datasets_dest = os.path.join(datastore_config['location'], 'datasets.csv')
-places_dest = os.path.join(datastore_config['location'], 'places.csv')
-summary_dest = os.path.join(datastore_config['location'], 'summary.csv')
-
-
-class AttrDict(dict):
-    __getattr__ = dict.__getitem__
-    __setattr__ = dict.__setitem__
-
-
-class Processor(object):
-    def download(self):
-        '''Download source data from the survey to tmp directory.'''
-        if not os.path.exists(config['tmp_path']):
-            os.makedirs(config['tmp_path'])
-        urllib.urlretrieve(survey_submissions, '{0}/submissions.csv'.format(config['tmp_path']))
-        urllib.urlretrieve(survey_entries, '{0}/entries.csv'.format(config['tmp_path']))
-        urllib.urlretrieve(survey_questions, '{0}/questions.csv'.format(config['tmp_path']))
-        urllib.urlretrieve(survey_datasets, '{0}/datasets.csv'.format(config['tmp_path']))
-        urllib.urlretrieve(survey_places, '{0}/places.csv'.format(config['tmp_path']))
-
-    def extract(self):
-        '''Extract data from raw Survey files, process and save to data dir'''
-        ext = Extractor()
-        ext.run()
-
-    def cleanup(self):
-        if os.path.exists('tmp'):
-            shutil.rmtree('tmp')
-
-    def run(self):
-        '''Run all stages of the processing pipeline'''
-        self.download()
-        self.extract()
-        # keep tmp around so we can run extract without download
-        # self.cleanup()
-
-    def test(self):
-        '''Run tests against extracted data to check all is good.'''
-        suite = unittest.TestLoader().loadTestsFromTestCase(TestIndexData)
-        unittest.TextTestRunner(verbosity=2).run(suite)
-
-
-class Extractor(object):
-    def __init__(self):
-        self.places = self._load_csv('tmp/places.csv')
-        self.entries = self._load_csv('tmp/entries.csv')
-        self.datasets = self._load_csv('tmp/datasets.csv')
-        self.questions = self._load_csv('tmp/questions.csv')
-        self.submissions = self._load_csv('tmp/submissions.csv')
-
-        self.current_year = '2014'
-        self.years = ['2014', '2013']
-        self.like_base_year = '2013' # the baseline for "like" comparisons
-        self.remove_places = ['tc', 'ae', 'ua', 'kn', 'vg', 'ye', 'bh', 'bs',
```

Comment:
@roll maybe i missed it... but do we have a place to control, per year, datasets and places that will be included. ideally we'd need both an include and exclude filter.